apps/docs/.vitepress/config.ts

```diff
@@ -381,6 +381,10 @@ export default defineConfig({
                 text: 'Optimizing Frontend Apps',
                 link: '/guide/transactions/optimizing-frontend-apps',
               },
+              {
+                text: 'Pre-confirmation',
+                link: '/guide/transactions/pre-confirmation',
```

Comment:
As I have already suggested in a couple of places, I have the impression it would be better to use plurals when speaking in general terms. See what you think.

```suggestion
                text: 'Pre-Confirmations',
                link: '/guide/transactions/pre-confirmations',
```